benchmarking/evaluations/metrics/envy_free_up_to_one_item.py

```diff
@@ -0,0 +1,66 @@
+import copy
+
+from api.models.team import Team
+from api.models.team_set import TeamSet
+from benchmarking.evaluations.interfaces import TeamSetMetric
+
+
+class EnvyFreenessUpToOneItem(TeamSetMetric):
+    """
+    Calculate the envy-free up to one metric for the given attribute for each team and average the results
+
+    A allocation is envy-free up to one if no agent envies another agent by more than one unit of the given attribute
+    Meaning if A is envying B, then A will no longer envy B if we remove one unit of the given attribute from  B
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(name="EnvyFreenessUpToOneItem", *args, **kwargs)
```

Comment:
if it has a theoretical range, include that here
Don't set the name here, it is intended that people can overwrite the name as needed